### DISCUSSION TITLE
Welcome to Feedland Blogroll WordPress Plugin Discussions!

### DISCUSSION THREAD

**Discussion by NickGreen** [a8cteam51/feedland-blogroll#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by scripting** [comment]
Greetings @NickGreen -- I have noted this group and will direct people here when they have questions about the Blogroll plugin for WordPress.
